src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,23 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  (comments.length > 0)
+    ? (
+      <div className="CommentList">
+        {comments.map(comment => (
+          <CommentInfo
+            comment={comment}
+            key={comment.id}
+          />
+        ))}
+      </div>
+    )
+    : (
+      <>
```

Comment:
It's better to wrap 'No comments yet' message in a semantic HTML tag like `<div>`, `<p>` or `<section>` for better accessibility and SEO.